packages/embark-ui/src/containers/ContractsContainer.js

```diff
@@ -1,25 +1,31 @@
 import React, {Component} from 'react';
 import {connect} from 'react-redux';
 import PropTypes from 'prop-types';
-import {contracts as contractsAction} from "../actions";
+import {
+  listenToContracts as listenToContractsAction,
+  contracts as contractsAction
+} from "../actions";
 
 import Contracts from '../components/Contracts';
+import ContractsList from '../components/ContractsList';
 import DataWrapper from "../components/DataWrapper";
 import PageHead from "../components/PageHead";
 import {getContracts} from "../reducers/selectors";
 
 class ContractsContainer extends Component {
   componentDidMount() {
     this.props.fetchContracts();
+    this.props.listenToContracts();
```

Comment:
Now that it is not in app container there is some chance that is container will be unmount.
In that case we should stop the container
=> feel free to do in another PR

Reply:
Nice catch @alaibe, updated!